src/scripts/iri_sources.py

```diff
@@ -0,0 +1,58 @@
+"""Get a list of version IRIs for sources"""
+import os
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Dict, List, Union
+
+from oaklib import get_adapter
+from oaklib.implementations import SqlImplementation
+
+HERE = Path(os.path.abspath(os.path.dirname(__file__)))
+SRC_DIR = HERE.parent
+PROJECT_ROOT = SRC_DIR.parent
+ONTO_DIR = PROJECT_ROOT / 'src' / 'ontology'
+
+
+
+def iri_docs(rel_paths: List[str], outpath_txt: Union[Path, str], outpath_md: Union[Path, str]):
+    """Get a list of version IRIs for sources"""
+    version_curies: List[str] = []
+    for rel_path in rel_paths:
+        path = ONTO_DIR / rel_path
+        adapter: SqlImplementation = get_adapter(path)
+        for ont in adapter.ontologies():
+            meta = adapter.ontology_metadata_map(ont)
+            versions: List[str] = meta["owl:versionIRI"]
```

Comment:
shouldn't `versions` be `version`, it returns one version from the ontology

Reply:
For whatever reason, it returns a list.